### ISSUE TITLE
Unable to docker-compose up 

### ISSUE THREAD

**Issue by leian-ivey** [d-led/gocd_docker_compose_example#1]
I get the following error: 

```
Pulling goagent_java (gocd/gocd-agent-ubuntu-16.04:latest)...
ERROR: manifest for gocd/gocd-agent-ubuntu-16.04:latest not found
```

**Comment by d-led** [comment]
the new docker containers should be up on Docker hub this week, then this will work again. I was surprised myself, as well

**Comment by d-led** [comment]
as soon as this repo contains a Dockerfile, and is built: https://github.com/gocd/docker-gocd-agent-ubuntu-16.04

**Comment by d-led** [comment]
@leian-ivey The [Dockerfiles](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-16.04) are up. This should work now

**Comment by d-led** [comment]
gotta switch to `gocd/gocd-agent-ubuntu-16.04:v17.3.0` (no latest tag yet)

**Comment by d-led** [comment]
will fix it tonight, as lots has changed in 17.3

**Comment by d-led** [comment]
rebuilt from scratch - works again
